### PR TITLE
Cleanup deprecated terms

### PR DESCRIPTION
**Pull request by yonghuah**

### CODE REVIEW

File: hypervisor/dm/vpic.c [hunk 1]
```diff
@@ -150,37 +150,37 @@ static void vpic_notify_intr(struct acrn_vpic *vpic)
 	uint32_t pin;
 
 	/*
-	 * First check the slave.
+	 * First check the Secondary.
```
Comment (dongyaozu) [inline hypervisor/dm/vpic.c:153]: secondary VPIC

File: hypervisor/dm/vpic.c [hunk 2]
```diff
@@ -150,37 +150,37 @@ static void vpic_notify_intr(struct acrn_vpic *vpic)
...
 	 */
 	i8259 = &vpic->i8259[1];
 	pin = vpic_get_highest_irrpin(i8259);
 	if (!i8259->intr_raised && (pin < NR_VPIC_PINS_PER_CHIP)) {
 		dev_dbg(DBG_LEVEL_PIC,
-		"pic slave notify pin = %hhu (imr 0x%x irr 0x%x isr 0x%x)\n",
+		"pic secondary notify pin = %hhu (imr 0x%x irr 0x%x isr 0x%x)\n",
```
Comment (dongyaozu) [inline hypervisor/dm/vpic.c:159]: Secondary PIC

File: hypervisor/dm/vpic.c [hunk 3]
```diff
@@ -150,37 +150,37 @@ static void vpic_notify_intr(struct acrn_vpic *vpic)
...
 		pin, i8259->mask, i8259->request, i8259->service);
 
 		/*
-		 * Cascade the request from the slave to the master.
+		 * Cascade the request from the Secondary to the Primary.
```
Comment (dongyaozu) [inline hypervisor/dm/vpic.c:163]: secondary PIC to primary PIC

File: hypervisor/dm/vpic.c [hunk 4]
```diff
@@ -150,37 +150,37 @@ static void vpic_notify_intr(struct acrn_vpic *vpic)
...
 		 */
 		i8259->intr_raised = true;
 		vpic_set_pinstate(vpic, 2U, 1U);
 		vpic_set_pinstate(vpic, 2U, 0U);
 	} else {
 		dev_dbg(DBG_LEVEL_PIC,
-		"pic slave no eligible interrupt (imr 0x%x irr 0x%x isr 0x%x)",
+		"pic secondary no eligible interrupt (imr 0x%x irr 0x%x isr 0x%x)",
```
Comment (dongyaozu) [inline hypervisor/dm/vpic.c:170]: ditto

**Comment by yonghuah** [comment]
@dongyaozu  have update this PR, please review, thanks.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Review by dongyaozu — APPROVED** [review]

**Pull request merged**